rabbit_monitor.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+#   coding: UTF-8
+import logging
+import sys
+from time import sleep
+
+import requests
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+from structlog import wrap_logger
+
+import settings
```

Comment:
@tundish can you put this down as an issue? I'll assign it a release milestone then if we decide that's the way we want to go.